lab-ken/test/greet-test.js

```diff
@@ -0,0 +1,37 @@
+'use strict';
+
+const chai = require('chai');
+const greet = require ('../lib/greet.js');
+const arithmetic = require ('../lib/arithmetic.js');
+
+const expect = chai.expect;
+
+describe ('testing module greet.js', function(){
+  it('should return "hello world"', function(){
+    expect(greet('world')).to.equal('Hello, world');
+  });
+  it ('should return null if value is not a string', function(){
+    expect(greet(false)).to.equal(null);
+  });
+});
+
+describe('testing module arithmetic.js', function(){
```

Comment:
My bad when I was helping you last night, I should have advised that your arithmetic module should be tested in its own test module, i.e. "arithmetic-test.js." Either way, this still works and I'm not docking points for it, just an FYI for the future.